common/options.go

```diff
@@ -10,6 +10,8 @@ const (
 	UrlParameterWithTransactions = "withTxs"
 	// UrlParameterWithLogs represents the name of an URL parameter
 	UrlParameterWithLogs = "withLogs"
+	// UrlParameterNotarizedAtSource represents the name of an URL parameter
+	UrlParameterNotarizedAtSource = "notarized-at-source"
```

Comment:
The other options follow the `camelCase` format. What should we do in this case?

Reply:
renamed to be camelCase